### ISSUE TITLE
CourierProvider missing children property with React 18

### ISSUE THREAD

**Issue by kevpie** [trycourier/courier-react#262]
Receiving this error `Property 'children' does not exist on type 'IntrinsicAttributes & ICourierProviderProps'` when using CourierProvider.

Looking for a work around if one exists or I can create a PR if given some guidance.  

Some advice online indicates to simply extending PropsWithChildren. 
I'm unsure how to keep that backwards with earlier versions of React.

Package `@trycourier/courier-provider`

**Comment by kevpie** [comment]
I used `patch-package` to get my codebase happy.
This is the patch I'm automatically applying and seems to do the trick. I'm satisfied with this workaround.

```patch
diff --git a/node_modules/@trycourier/react-provider/typings/types.d.ts b/node_modules/@trycourier/react-provider/typings/types.d.ts
index 4fa29ca..fe06414 100644
--- a/node_modules/@trycourier/react-provider/typings/types.d.ts
+++ b/node_modules/@trycourier/react-provider/typings/types.d.ts
@@ -1,6 +1,7 @@
 import { Transport } from "./transports";
 import { Interceptor } from "./transports/types";
 import { ErrorEvent } from "reconnecting-websocket";
+import { PropsWithChildren } from "react";
 export declare type ErrorEventHandler = (event: ErrorEvent) => void;
 export declare type WSOptions = {
     url?: string;
@@ -36,7 +37,7 @@ export interface Brand {
         tertiary?: string;
     };
 }
-export interface ICourierProviderProps {
+export interface ICourierProviderProps extends PropsWithChildren {
     apiUrl?: string;
     brand?: Brand;
     brandId?: string;
```

**Comment by fjflala** [comment]
Hello is there any news regarding this.
Do we have any estimated time to upgrade the React version to 18? 
Happy to help if needed.

**Comment by rileylnapier** [comment]
Hi, working on this right now.  Sorry we've been relying on our slack channel for support and ill make sure I have notifications turned on for issues here too.

**Comment by rileylnapier** [comment]
hi, im taking a look here, when i tested it before with CRA i assumed it was working with react 18 but that wasn't ts so i missed this:  https://github.com/trycourier/courier-react/pull/441

**Comment by rileylnapier** [comment]
hi, i just released 3.6.3 ~ sorry for the delay.  ill make sure to check in on the github issues from now on.
